extension/src/popup/views/AddAccount/AddAccount/index.tsx

```diff
@@ -46,6 +50,12 @@ export const AddAccount = () => {
     }
   };
 
+  React.useEffect(() => {
```

Comment:
Just to match style - I'd `import {useEffect} from "react";`

and just call `useEffect` here

Reply:
yup, will do